src/sql/azdata.d.ts

```diff
@@ -3171,10 +3171,10 @@ declare module 'azdata' {
 
 	export interface DeclarativeTableColumn {
 		displayName: string;
-		categoryValues: CategoryValue[];
 		valueType: DeclarativeDataType;
 		isReadOnly: boolean;
 		width: number | string;
+		categoryValues?: CategoryValue[];
```

Comment:
This was already able to handle an undefined categoryValues so I'm updating it directly since functionally nothing is changing - it's just the signature.